activiti-spring-connector/src/test/java/org/activiti/spring/connector/EmptyConnectorServiceIT.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.spring.connector;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.activiti.model.connector.Connector;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@TestPropertySource(locations = "classpath:application-empty-test.properties")
+public class EmptyConnectorServiceIT {
+
+    @Autowired
+    private ConnectorService connectorService;
+
+    @Test
+    public void emptyConnectors() throws IOException {
+
+        List<Connector> connectors = connectorService.get();
+        assertNotNull(connectors);
+        assertEquals(0, connectors.size());
+    }
+
+    @org.springframework.context.annotation.Configuration
```

Comment:
@ffazzini do we need the FQN here? 
Also this highlight the need for an AutoConfiguration, can we make sure that we add one?

Reply:
If it is possible i would just use autoconfiguration rather than componentscan, about annotation or not i don't see much improvement as it still needs manual import by the dev. I would like something more transparent like an autoconfiguration. I will investigate how to do that